### DISCUSSION TITLE
Multiple sites with the same domain

### DISCUSSION THREAD

**Discussion by anderius** [plausible/analytics#934] [Anything Else]
We have multiple applications that we want to analyze, all served on the same domain with different paths.

It would be nice if we could configure each application with some sort if plausible id, instead of the hostname (that would be the same).

Example:

example.org/app1
example.org/app2

Etc.

**Comment by Vigasaurus** [comment]
The domain you give to Plausible doesn't actually have to be the exact domain you'll be sending events from, it really is more like an ID in the format of a hostname. 

So you could for example, do
app1.mydomain.com
app2.mydomain.com 

as the sites in plausible (even if these subdomains are meaningless), and just send the events as appropriate with the matching data-domain.

**Reply by anderius** [reply]
That was a bit confusing, but I see there is a separate feature request for replacing the domain with another identifier.

Thank you for this clarification! :-)

**Comment by anderius** [comment]
I don't know this part of GitHub (discussions), but it seems a feature request can not be marked "completed" by the requester. Well, those who can, feel free! :-)
